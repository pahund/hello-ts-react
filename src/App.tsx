import React from "react";
import AceEditor from "react-ace";

export default () => (
  <div>
    <p>Hello world!</p>
    <AceEditor mode="javascript" theme="github" name="editor" value="// Hai!" />
  </div>
);
