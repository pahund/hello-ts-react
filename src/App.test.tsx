import React from "react";
import App from "./App";

test("loads and displays the app", () => {
  expect(<App />).toMatchSnapshot();
});
