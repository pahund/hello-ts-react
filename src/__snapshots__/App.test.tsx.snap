// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loads and displays the app 1`] = `<_default />`;
